tests/integration-templates/tests/generate-tests.ts

```diff
@@ -0,0 +1,82 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { fileURLToPath } from 'url';
+import ejs from 'ejs';
+import yaml from 'js-yaml';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const fullPath = process.cwd();
+const configPath = path.resolve(fullPath, './nango-integrations/nango.yaml');
+const config: any = yaml.load(fs.readFileSync(configPath, 'utf8'));
+const { integrations } = config;
+
+for (const integration in integrations) {
+    const { syncs } = integrations[integration];
+    for (const syncName in syncs) {
+        const sync = syncs[syncName];
+        generateSyncTest(integration, syncName, sync.output);
+    }
+}
+
+function generateSyncTest(integration: string, syncName: string, modelName: string) {
+    const data: Record<string, string | Record<string, string | boolean>> = {
+        integration,
+        syncName,
+        modelName
+    };
+
+    if (fs.existsSync(path.resolve(__dirname, `../nango-integrations/${integration}/mocks/${syncName}/batchDelete.json`))) {
```

Comment:
(comment applicable for a lot of places) storing the base file path would be great for readability

Reply:
Updated